pkg/logql/log/parser_test.go

```diff
@@ -720,6 +720,88 @@ func Test_logfmtParser_Parse(t *testing.T) {
 	}
 }
 
+func Test_syslogParser_Parse(t *testing.T) {
+	tests := []struct {
+		name string
+		line []byte
+		lbs  labels.Labels
+		want labels.Labels
+	}{
+		{
+			"not syslog",
+			[]byte("not a syslog"),
+			labels.Labels{
+				{Name: "foo", Value: "bar"},
+			},
+			labels.Labels{
```

Comment:
Q: passing random string to syslog `parser.Process()` returning no error? sounds bit weird. is it expected?

Reply:
You're right, this should add a syslog error to the labels.